lib/graph3d/Range.js

```diff
@@ -0,0 +1,85 @@
+////////////////////////////////////////////////////////////////////////////////
+// class Range
+//
+// Helper class to make working with related min and max values easier.
+//
+//
+// The range is inclusive; a given value is considered part of the range if:
+//
+//    this.min <= value <= this.max
+//
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
You should use the official jsdoc header style. Here is an example: https://github.com/almende/vis/blob/develop/lib/graph3d/StepNumber.js#L1